### PR TITLE
some care about the documentation of graph generators

### PR DESCRIPTION
**Pull request by dcoudert**
We fix various links, use of math mode, etc.

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: src/sage/graphs/generators/random.py [hunk 1]
```diff
@@ -1733,16 +1733,19 @@ def RandomToleranceGraph(n, seed=None):
...
+    interval and `t_i` a random positive value. The width of the representation
+    is limited to `n^2 * 2^n`.
 
     .. NOTE::
 
-        The vertices are named 0, 1, ..., n-1. The tolerance representation used
-        to create the graph is saved with the graph and can be recovered using
-        ``get_vertex()`` or ``get_vertices()``.
+        The vertices are named `0, 1, \cdots, n-1`. The tolerance representation
+        used to create the graph is saved with the graph and can be recovered
```
Comment (kwankyu) [inline src/sage/graphs/generators/random.py:1746]: The dots should be lower. Hence (1)
```
`0, 1, \ldots, n-1`
```
or (2)
```
`0, 1, \dots, n-1`
```
or (3)
```
`0`, `1`, ..., `n-1`
```
Personally (2) looks best.

File: src/sage/graphs/generators/families.py [hunk 2]
```diff
@@ -2223,19 +2221,18 @@ def LCFGraph(n, shift_list, repeats):
...
-    i + j\*len(shift_list) + s_i).
+    LCF (Lederberg-Coxeter-Fruchte) notation is a concise way of describing
+    cubic Hamiltonian graphs. The way a graph is constructed is as
+    follows. Since there is a Hamiltonian cycle, we first create a cycle on `n`
+    nodes. The variable `\textrm{shift_list} = [s_0, s_1, ..., s_k-1]`
+    describes edges to be created by the following scheme: for each `i`, connect
+    vertex `i` to vertex `(i + s_i)`. Then, repeats specifies the number of
+    times to repeat this process, where on the `j`-th repeat we connect vertex
+    `(i + j |\textrm{shift_list}|)` to vertex
+    `( i + j |\textrm{shift_list}| + s_i)`.
```
Comment (kwankyu) [inline src/sage/graphs/generators/families.py:2237]: As I understand this, `shift_list` and `repeats` should be code. Thus
```
The variable ``shift_list`` = `[s_0, s_1, ..., s_{k-1}]`
```
`s_k-1` seems an error. Hence the length of `shift_list` is `k`. Hence
```
Then, ``repeats`` specifies the number of
```
and
```
we connect vertex `i + jk` to vertex `i +jk + s_i`.
```
where i runs through 0, 1, ..., k - 1.

File: src/sage/graphs/generators/families.py [hunk 3]
```diff
@@ -787,14 +786,14 @@ def LollipopGraph(n1, n2):
 
 def TadpoleGraph(n1, n2):
     r"""
-    Return a tadpole graph with n1+n2 nodes.
+    Return a tadpole graph with ``n1+n2`` nodes.
```
Comment (kwankyu) [inline src/sage/graphs/generators/families.py:791]: I think it is okay to write
```
`n_1 + n_2`
```
as math. Also below.

File: src/sage/graphs/generators/families.py [hunk 4]
```diff
@@ -714,14 +713,14 @@ def BarbellGraph(n1, n2):
 
 def LollipopGraph(n1, n2):
     r"""
-    Returns a lollipop graph with n1+n2 nodes.
+    Returns a lollipop graph with ``n1+n2`` nodes.
 
-    A lollipop graph is a path graph (order n2) connected to a complete
-    graph (order n1). (A barbell graph minus one of the bells).
+    A lollipop graph is a path graph (order ``n2``) connected to a complete
+    graph (order ``n1``). (A barbell graph minus one of the bells).
```
Comment (kwankyu) [inline src/sage/graphs/generators/families.py:721]: The same comment as below.

File: src/sage/graphs/generators/families.py [hunk 5]
```diff
@@ -454,15 +453,15 @@ def HammingGraph(n, q, X=None):
         sage: g.is_vertex_transitive()                                                  # needs sage.groups
         True
 
-    A Hamming graph with parameters (1,q) is isomorphic to the
-    Complete graph with parameter q.  ::
+    A Hamming graph with parameters ``(1,q)`` is isomorphic to the
+    Complete graph with parameter ``q``::
```
Comment (kwankyu) [inline src/sage/graphs/generators/families.py:458]: Why not math?

File: src/sage/graphs/generators/random.py (hunk 1 above)
Comment (dcoudert) [reply src/sage/graphs/generators/random.py:1746]: I used (2)

File: src/sage/graphs/generators/families.py (hunk 2 above)
Comment (dcoudert) [reply src/sage/graphs/generators/families.py:2237]: I tried to improve it.

File: src/sage/graphs/generators/families.py (hunk 3 above)
Comment (dcoudert) [reply src/sage/graphs/generators/families.py:791]: I change that in several places.

File: src/sage/graphs/generators/families.py (hunk 4 above)
Comment (dcoudert) [reply src/sage/graphs/generators/families.py:721]: done

File: src/sage/graphs/generators/families.py (hunk 5 above)
Comment (dcoudert) [reply src/sage/graphs/generators/families.py:458]: changed.

**Review by kwankyu — APPROVED** [review]
Thanks. LGTM.

**Comment by dcoudert** [comment]
Thank you for the review. The documentation looks much better with this PR.

**Pull request merged**